Changelog.md

```diff
@@ -1,5 +1,12 @@
 ### Development
-[Full Changelog](http://github.com/rspec/rspec-rails/compare/v3.7.0...master)
+[Full Changelog](http://github.com/rspec/rspec-rails/compare/v3.7.1...master)
+
+### 3.7.1 / 2017-10-18
+[Full Changelog](http://github.com/rspec/rspec-rails/compare/v3.7.0...v3.7.1)
+
+Bug Fixes:
+
+* Fix puma loading in Rails 5.1 (Sam Phippen, #1884)
```

Comment:
A more descriptive change would be `Prevent system test integration loading when puma or capybara are missing`

Reply:
Sure thing, pushed an amended version with this change.